src/Selenium2Library/keywords/_javascript.py

```diff
@@ -129,19 +129,55 @@ def get_alert_message(self):
         """
         return self._close_alert()
 
+    def read_alert_message(self):
+        """ Returns the text of current JavaScript alert without closing it.
+
+        This keyword will fail if no alert is present. Note that
+        following keywords will fail unless the alert is
+        dismissed by this keyword or another like `Get Alert Message`.
+        """
+        return self._read_alert()
+
+    def handle_alert(self, confirm=False):
```

Comment:
This keyword I'm fine with. We need an explicit keyword for dealing with alerts anyway.